api/grpcserver.go

```diff
@@ -85,6 +87,14 @@ func init() {
 	prometheus.MustRegister(_apiCallSourceWithOutChainIDMtc)
 }
 
+// RecoveryInterceptor handles panic to a custom error
+func RecoveryInterceptor() grpc_recovery.Option {
+	return grpc_recovery.WithRecoveryHandler(func(p interface{}) (err error) {
+		recovery.CrashLog(p)
+		return grpc.Errorf(codes.Unknown, "panic triggered: %v", p)
```

Comment:
not sure about `codes.Unknown` is appropriate or not, but the error message should be updated. suggested message:"something goes wrong"